src/test/kotlin/com/pubnub/contract/member/step/ThenSteps.kt

```diff
@@ -13,12 +13,18 @@ class ThenSteps(
     fun the_response_contains_list_with_and_members(firstMemberName: String, secondMemberName: String) {
```

Comment:
There was an idea to replace : 
"And the response contains list with 'VipChatMembership' and 'DMMembership' memberships"
with two simpler steps:
And the response contains list with 'VipChatMembership' membership 
And the response contains list with 'DMMembership' membership

what about the idea?